__tests__/feature/dashboard.test.js

```diff
@@ -28,7 +28,7 @@ describe("viewing the dashboard", () => {
       await screen.findByText(/Uber template1/)
 
       // Click the resource template
-      fireEvent.click(screen.getByTitle("Create resource for Uber template1"))
+      fireEvent.click(screen.getByTestId("Create resource for Uber template1"))
```

Comment:
Why are you switching to `*ByTestId`.  I believe the guide says that we should avoid using this when possible.

Reply:
> In the spirit of the guiding principles, it is recommended to use this only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible. That said, they are way better than querying based on DOM structure or styling css class names. Learn more about data-testids from the blog post "Making your UI tests resilient to change"

https://testing-library.com/docs/queries/bytestid/

Reply:
It does. And even better, we should using `*ByRole`. However, these other queries are _very slow_, leading to _very slow_ and _very flaky_ tests. As a matter of convention, we have added `test-id` attributes to mirror other attributes, usually `aria-label`. Due to this convention, querying by test id is roughly equivalent to these better tests but much faster.

In this particular case, the title was changed to be consistent with other actions which had generic titles like "Create new" and specific arial-labels and test-ids like "Create resource for Uber template1".